lib/datasource/go/releases-direct.ts

```diff
@@ -103,5 +105,5 @@ export async function getReleases(
     );
   }
 
-  return res;
+  return { ...res, sourceUrl };
```

Comment:
Not necessary, but just for consistency

Reply:
Actually, I've called it here just to be sure tests are producing same results, like if `sourceUrl` from `res` is used